packages/analytics/src/get-config.ts

```diff
@@ -20,18 +20,41 @@
  */
 
 import { FirebaseApp } from '@firebase/app-types';
-import { DynamicConfig, ThrottleMetadata } from '@firebase/analytics-types';
+import {
+  DynamicConfig,
+  ThrottleMetadata,
+  MinimalDynamicConfig
+} from '@firebase/analytics-types';
 import { FirebaseError, calculateBackoffMillis } from '@firebase/util';
 import { AnalyticsError, ERROR_FACTORY } from './errors';
 import { DYNAMIC_CONFIG_URL, FETCH_TIMEOUT_MILLIS } from './constants';
 import { logger } from './logger';
 
+// App config fields needed by analytics.
+export interface AppFields {
+  appId: string;
+  apiKey: string;
+  measurementId?: string;
+}
+
+/**
+ * Initial retry interval in seconds for 503 errors, which we want to be conservative about
+ * to avoid overloading servers.
+ */
+export const LONG_RETRY_FACTOR = 30;
```

Comment:
It's not actually seconds, it just comes out to it because the default interval is 1 second. It's the base for the exponent in `calculateBackoffMillis`. I'll change the comment to be more accurate.

Reply:
Ah that makes sense.